README.md

```diff
@@ -52,14 +52,36 @@ Note: Be sure that the target Google Drive folder has edit access granted to the
 
 #### Building
 1. Start the database in a Docker container by running `docker-compose up` in a terminal.
-2. In a different terminal, execute the following command : `micronaut_environments=local ./gradlew build`
-      On Windows use `set "MICRONAUT_ENVIRONMENTS=local" `
-         then `gradlew build`
-3. Once the build is successful, execute: `./gradlew assemble && micronaut_environments=local ./gradlew run`
-      On Windows use 'gradlew assemble`
-         then `set MICRONAUT_ENVIRONMENTS=local`
-         then 'gradlew run' 
+2. In a different terminal, execute the following commands : 
+    * On Bash/Zsh -
+        ```sh
+        $ OAUTH_CLIENT_ID=<Insert_Client_ID> OAUTH_CLIENT_SECRET=<Insert_Client_Secret> MICRONAUT_ENVIRONMENTS=local ./gradlew build
+        ```
+        ```sh
+        $ ./gradlew assemble
+        ```
+        ```sh
+        $ OAUTH_CLIENT_ID=<Insert_Client_ID> OAUTH_CLIENT_SECRET=<Insert_Client_Secret> MICRONAUT_ENVIRONMENTS=local ./gradlew run
+        ```
+    
+    * On Powershell -
+        ```sh
+        $ set "MICRONAUT_ENVIRONMENTS=local"
```

Comment:
shouldn't the OAUTH variables also be represented for windows folk?

Reply:
It turns out that they need to set as Env Variables. Updated PR.